libwasmvm/build.rs

```diff
@@ -5,5 +5,5 @@ fn main() {
 
     cbindgen::generate(crate_dir)
         .expect("Unable to generate bindings")
-        .write_to_file("./api/bindings.h");
```

Comment:
It appears that `./api/bindings.h` has been moved to `./bindings.h`.
However, in this PR,
1. it seems that `./bindings.h` is newly created rather than move (any other file path changes are LGTM)
2. There is also have a fix in `./api/bindings.h`

Is there a reason why `./api/bindings.h` remains, and why is it newly created instead of move?

Reply:
This is because moving of rust dir (to libwasmvm). `bindings.h` can be created only under the libwasmvm dir. So, after rust dir moved, this is created as `./libwasmvm/bindings.h` first and then then it is copied to `./api/bindngs.h`

Reply:
I read the your description and found the below. I missed this. thks!
https://github.com/line/wasmvm/blob/f804f7309704f236fefa4fc712eb2320a700af58/Makefile#L89